src/domain/header/MainNavbarContainer.js

```diff
@@ -9,6 +9,11 @@ import MainNavbar from './MainNavbar';
 export const selector = createStructuredSelector({
   isAdmin: isAdminSelector,
   isLoggedIn: isLoggedInSelector,
+  respaAdminUrl: () => {
```

Comment:
With this assignment `respaAdminUrl` will be a function that returns a string and not just a string.

Reply:
Ah actually its the return value from the function that will be the type while using selector.

Reply:
Yea but when is the function called? As [here](https://github.com/andersinno/varaamo/pull/53/files#diff-f7b6691ae5a9a4525951600e3af070e73c6cb040ac09734c7ca0be2c6a2fffd9R83) you are just passing the function to the `NavItem` component.